src/components/LoadingSpinnerMap.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { ActivityIndicator, StyleSheet, View } from 'react-native';
+
+import { colors } from '../config';
+
+type Props = {
+  loading?: boolean;
+};
+
+export const LoadingSpinnerMap = ({ loading }: Props) => {
```

Comment:
what is the idea behind this component? what needs to be waited for? and what is the benefit of "seeing through the loading component"?

Reply:
we need to wait for the map to finish loading, loading the markers, clusters and the correct mapview regarding the clusters and markers set on the map.

the see through is mainly for dev reasons so we can check the loading times exactly once we have this new map feature in our dev build to test it on various devices. 
it's also psychologically beneficiary for us cause the user is willing to wait a second longer if they see that something is happening wanting to see the result of the finished map instead of just waiting for the plain loadingspinner to stop spinning.

Reply:
I updated the `LoadingContainer` component by adding `containerStyle` prop and removed this component. I don't think we need an extra component. 

I also shortened the wait times. As far as I noticed, this loading indicator disappears after 4 seconds of setTimout. We had an extra 4 seconds of waiting time every time the map view was opened. The 4 seconds does not check if the map is loaded or not.